packages/frontend/src/components/ChartConfigPanel/Series/Series.styles.ts

```diff
@@ -60,12 +60,21 @@ export const SeriesBlock = styled.div`
     flex-direction: column;
 `;
 
-export const SeriesWrapper = styled.div`
+export const SeriesWrapper = styled.div<{ $isSingle?: boolean }>`
     ${FlexTemplate}
 
     &:last-child {
         margin-bottom: 0;
     }
+
+    ${({ $isSingle }) =>
```

Comment:
Maybe I'm wrong, but I find this syntax a bit weird, should we just use a different class name? , eg: `series-wrapper-single` (not sure if that would make it more readable)